src/lib/modules/SearchControls/SearchAggregationsMenu.js

```diff
@@ -6,52 +6,22 @@ import { getSearchConfig } from '@config';
 import PropTypes from 'prop-types';
 
 export default class SearchAggregationsMenu extends Component {
-  constructor(props) {
-    super(props);
-    this.state = { menuOpen: false };
-  }
-
-  toggleMenuOpen = (e, elem) => {
-    const { menuOpen } = this.state;
-    this.setState({ menuOpen: !menuOpen });
-  };
-
-  closeMenu = (e, elem) => {
-    this.setState({ menuOpen: false });
-  };
-
   render() {
     const { modelName } = this.props;
-    const { menuOpen } = this.state;
     const searchConfig = getSearchConfig(modelName);
-    // this element is needed to prevent the Dropdown menu
```

Comment:
did you check if this is still the issue after your change? 
<img width="183" alt="Screenshot 2021-01-06 at 15 09 17" src="https://user-images.githubusercontent.com/38131488/103777461-318ee500-5031-11eb-928d-3a723801b93e.png">
the problem was that when you clicked on 'Literature types" or any other category of filters the whole menu was collapsing, and choosing a filter was not possible

Reply:
this problem is fixed